src/couch_mrview_updater.erl

```diff
@@ -467,3 +469,9 @@ update_task(NumChanges) ->
             (Changes2 * 100) div Total
     end,
     couch_task_status:update([{progress, Progress}, {changes_done, Changes2}]).
+
+
+maybe_notify(State, View, KVs, ToRem) ->
+    Updated = [Key || {{Key, _}, _} <- KVs],
```

Comment:
How big KVs and ToRem are here? Is it possible and reasonable to LBYL and check if anyone is actually waits for this notification?